pkg/knative/deployer_int_test.go

```diff
@@ -1,858 +1,60 @@
 //go:build integration
+// +build integration
```

Comment:
there is one again. Is there a reason why only some contain this?

Reply:
I guess this was simply some copy&paste mistake. Will update it